recipes/mosdepth/build.sh

```
@@ -1,20 +1,24 @@
-#!/bin/sh
+#!/bin/bash
 
-if [[ ${target_platform}  == osx-64 ]] ; then
-    curl -SL https://github.com/nim-lang/nightlies/releases/download/latest-version-1-6/macosx_x64.tar.xz -o macosx_x64.tar.xz
+if [[ ${target_platform}  == "osx-64" ]] ; then
+    curl -SL https://github.com/nim-lang/nightlies/releases/download/latest-version-2-2/macosx_x64.tar.xz -o macosx_x64.tar.xz
     tar -xzf macosx_x64.tar.xz
-    cd nim-1.6.*
+    cd nim-2.2.*
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)